db/schema.rb

```diff
@@ -14,7 +14,7 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "dblink"
+  enable_extension "pg_stat_statements" unless Rails.env.test?
```

Comment:
Er should this be `enable_extension "dblink" unless Rails.env.test?`

Reply:
Yeah, it definitely should be, thanks!